make/test.mk

```diff
@@ -46,12 +46,14 @@ upload-codecov-report:
 ###########################################################
 
 .PHONY: test-e2e
-test-e2e:  e2e-setup
-	sed -e 's|REPLACE_IMAGE|${IMAGE_NAME}|g' ./deploy/operator.yaml  | oc apply -f -
+test-e2e:  deploy-host e2e-setup setup-kubefed
 	# This is hack to fix https://github.com/operator-framework/operator-sdk/issues/1657
 	echo "info: Running go mod vendor"
 	go mod vendor
 	operator-sdk test local ./test/e2e --no-setup --namespace $(TEST_NAMESPACE) --go-test-flags "-v -timeout=15m"
+	# remove me once verified
+	oc get kubefedcluster -n $(TEST_NAMESPACE)
+	oc get kubefedcluster -n $(HOST_NS)
```

Comment:
do we still need them?

Reply:
I was thinking to remove it, but later on realized it would be useful to debug, if something goes wrong, at-least you would be sure about cluster is syncing, otherwise we need to wait for another more 30 minutes to verify it.